include/rviz_visual_tools/remote_control.hpp

```diff
@@ -51,22 +53,21 @@ class RemoteControl
 {
 public:
   /**
-   * \brief Constructor for passing in NodeOptions and an executor.
-   * \param node - A node that will be added to an executor outside of this class
+   * \brief Constructor for passing in NodeOptions
+   * \param node - A node
    */
-  explicit RemoteControl(const rclcpp::Executor::SharedPtr& executor,
-                         const rclcpp::NodeOptions& node_options)
-    : RemoteControl(executor, std::make_shared<rclcpp::Node>("remote_control", node_options))
+  explicit RemoteControl(const rclcpp::NodeOptions& node_options)
```

Comment:
> Old constructor wasn't working and I am guessing noone was using it as well. I can make it work and put a deprecation warning but I feel that would be a bit unintuitive since it should not be used that way as of foxy.

That's the point of deprecating it. The old constructor could simply call the new constructor and ignore the executor, that would work.

Reply:
Sure on it.